tee-worker/service/src/ocall_bridge/worker_on_chain_ocall.rs

```diff
@@ -44,10 +44,12 @@ impl<E, F> WorkerOnChainOCall<E, F> {
 
 impl<E, F> WorkerOnChainBridge for WorkerOnChainOCall<E, F>
 where
-	E: EnclaveBase,
-	F: CreateNodeApi,
+	E: EnclaveBase + std::marker::Send + std::marker::Sync,
+	F: CreateNodeApi + std::marker::Send + std::marker::Sync + 'static,
```

Comment:
Why do we need this?

Reply:
Caused by build error: `F` cannot be sent between threads safely.

Reply:
You shouldn't need this tho - or why didn't we need that? What has changed?